app/assets/javascripts/browsered/web-payments/index.js

```diff
@@ -1,33 +1,41 @@
 'use strict'
 
+// Local dependencies
 const { clearErrorSummary } = require('./helpers')
 const makeApplePayRequest = require('./apple-pay')
 const { createGooglePaymentRequest, googlePayNow } = require('./google-pay')
 
-module.exports = () => {
-  const paymentMethodForm = document.getElementById('payment-request-container')
-  const standardMethodContainer = document.getElementById('enter-card-details-container')
-
-  if (window.PaymentRequest && paymentMethodForm) {
-    if (window.ApplePaySession && ApplePaySession.canMakePayments()) {
-      paymentMethodForm.classList.remove('hidden')
-      standardMethodContainer.classList.add('hidden')
-      document.getElementById('payment-method-apple-pay').parentNode.style.display = 'block'
-      document.getElementById('payment-method-apple-pay').checked = true
-    } else {
-      console.log('payment request API is available')
-      const request = createGooglePaymentRequest();
-      request.canMakePayment()
-        .then(result => {
-          if (result) {
-            console.log('Google Pay is available')
-            paymentMethodForm.classList.remove('hidden')
-            standardMethodContainer.classList.add('hidden')
-            document.getElementById('payment-method-google-pay').parentNode.style.display = 'block'
-          }
-        })
-    }
+// Browser elements
+const paymentMethodForm = document.getElementById('payment-request-container')
+const standardMethodContainer = document.getElementById('enter-card-details-container')
 
+const initApplePayIfAvailable = () => {
+  if (window.ApplePaySession && ApplePaySession.canMakePayments()) {
+    paymentMethodForm.classList.remove('hidden')
+    standardMethodContainer.classList.add('hidden')
+    document.getElementById('payment-method-apple-pay').parentNode.style.display = 'block'
+    document.getElementById('payment-method-apple-pay').checked = true
+  }
+}
+
+const initGooglePayIfAvailable = () => {
+  if (window.PaymentRequest) {
+    console.log('payment request API is available')
```

Comment:
I guess we shouldn't leave these `console.log` instances in production, could we at least add a `TODO` to remove them before we go live?

Reply:
Yeah I left them all in because it’s all behind the feature flag and there is still a few bits to do to prepare this for production around getting vars and stuff, I guess wouldn’t do any harm if I added TODOs to help make it clear I will remove